.github/workflows/ci.yml

```diff
@@ -50,7 +50,7 @@ jobs:
             python-version: 3.8
             toxenv: py38
 
-          - name: jwst.datamodels tests
+          - name: jwst tests
```

Comment:
Is the benefit of running all the jwst tests worth the cost?  Without this the test suite takes about a minute.

Reply:
Good question.  The reason we only ran `jwst.datamodels` before is that that submodule was the only one that could run with `--pyargs` when we set up this job eons ago.  Now the whole suite can run with `--pyargs`, so, perhaps better coverage is  better?  That said, I don't know if it actually is better coverage of the code here.  Hmm.